### DISCUSSION TITLE
Is icub website off?

### DISCUSSION THREAD

**Discussion by tasbolat1** [robotology/community#294] [Q&A] [answered]
Dear icub community,
It's been several days that I cannot access icub's website. For example the following link http://wiki.icub.org/brain/group__icub__skinManager.html gives me an error 404. And, many other pages also give that error. Has website moved to another domain/server? 

Thanks,
Tasbolat

**Comment by pattacini** [comment, marked as answer]
The correct URL is http://www.icub.org/doc/icub-main/group__icub__skinManager.html, as retrieved by visiting the manual main page (http://www.icub.org/doc/icub-main/group__icub__modules__all.html).

Probably, you found yourself in the wrong web page through a google search, which picked up an old resource.

**Comment by tasbolat1** [comment]
Thanks @pattacini for the link. Now docs/manuals/tutorials are accessible.

Best,
Tasbolat
